src/services.js

```diff
@@ -78,6 +78,19 @@ export function login() {
   );
 }
 
+export async function pullFriendRequests(spotifyId) {
+  return fetch(`/friend/requests?id=${spotifyId}`).then(
+    async (data) => {
+      const friendRequests = await data.json();
+      if (!friendRequests) return;
+      store.dispatch(setFriendRequests(friendRequests));
+    },
+    (err) => {
+      console.log("Something went wrong ", err);
```

Comment:
Can we add [ERROR] in front of the errors in logging, going to add it in front of the rest of them later tonight

Reply:
> lgtm. I couldn't do a lot of testing because I have everyone added already, I was able to add myself though, do we want to limit this?

Let me throw in a check in the backend for that.

Reply:
For logging, i'll work on a presentable solution instead - we shouldn't be console logging errors in prod anyways. We can do that tags, but I'm going to work on having their verbosity dependent on whether `.env` includes a prod tag.